widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/RemoteConfiguration.kt

```diff
@@ -26,14 +27,18 @@ internal data class RemoteConfiguration(
     val alertRemoteConfig: AlertRemoteConfig?,
 
     @SerializedName("endScreenSharing")
-    val endScreenShareRemoteConfig: EndScreenShareRemoteConfig?
+    val endScreenShareRemoteConfig: EndScreenShareRemoteConfig?,
```

Comment:
FYI: you will probably have merge conflicts. In another PR (should be already merged) moved the `endScreenShareRemoteConfig` under `callVisualizerRemoteConfig`

Reply:
Thank you for informing